aws-api/src/main/java/com/amplifyframework/api/aws/AppSyncGraphQLOperation.java

```diff
@@ -142,6 +142,9 @@ public void onResponse(@NonNull Call call, @NonNull Response response) {
                     return;
                 }
             }
+            if(response.code() >= 400) {
```

Comment:
When you merge the latest commit from main, make sure to add this block in the `MultiAuthAppSyncOperation` class as well.

Reply:
Would it be possible to centralize this logic into one place that can be referenced from `AppSyncGraphQLOperation` and `MultiAuthAppSyncOperation`?

Reply:
Created a task for this one.